_docs_use-cases/use-case-consumption-share-a-repositoryattribute.md

```diff
@@ -63,9 +63,9 @@ If you have already shared another version of a succeeded Attribute with the pee
 
 ## On Failure
 
-- The Request cannot be created, if the `peer` is unknown.
-- The Request cannot be created, if the `attributeId` belongs to a RelationshipAttribute.
-- The Request cannot be created, if the `attributeId` belongs to an IdentityAttribute with a `shareInfo`.
-- The Request cannot be created, if the Attribute has already been shared with the peer.
-- The Request cannot be created, if another version of the Attribute regarding succession has already been shared with the peer.
-- The Request cannot be created, if the parameters are malformed.
+- The Request cannot be created if the `peer` is unknown.
+- The Request cannot be created if the `attributeId` belongs to a RelationshipAttribute.
+- The Request cannot be created if the `attributeId` belongs to an IdentityAttribute with a `shareInfo`.
+- The Request cannot be created if the Attribute has already been shared with the peer and the own shared IdentityAttribute doesn't have `"DeletedByPeer"` or `"ToBeDeletedByPeer"` as `deletionInfo.deletionStatus`.
+- The Request cannot be created if another version of the Attribute regarding succession has already been shared with the peer, unless the latest shared version has "DeletedByPeer" or "ToBeDeletedByPeer" as `deletionInfo.deletionStatus`.
```

Comment:
Use formatting with backticks for "DeletedByPeer" and "ToBeDeletedByPeer" as you do elsewhere please.